server/utils/security/command_safety.py

```
@@ -0,0 +1,241 @@
+        future = _executor.submit(llm.invoke, messages)
+        result = future.result(timeout=_TIMEOUT_SECONDS)
+    except Exception as e:
+        error_msg = str(e)
+        raise
+    finally:
+        _track_usage(user_id, session_id, messages, start, error_msg)
+
+    return result if isinstance(result, SafetyVerdict) else SafetyVerdict.model_validate(result)
+
+
+def _track_usage(user_id, session_id, messages, start, error_msg):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_track_usage`

Add return type annotation: `None`

(ANN202)